### PR TITLE
Bump minimum Hugo version in README

### PR DESCRIPTION
**Pull request by zecakeh**
To match the one in `config.toml` changed in #1775.

Preview: https://pr1788--matrix-spec-previews.netlify.app

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -61,7 +61,7 @@ place after an MSC has been accepted, not as part of a proposal itself.
 
 1. Install the extended version (often the OS default) of Hugo:
    <https://gohugo.io/getting-started/installing>. Note that at least Hugo
-   v0.110.0 is required.
+   v0.113.0 is required.
```
Comment (richvdh) [inline README.md:64]: Is it though? As I said in the chat, there's no need to bump this unless we're aware of problems on v0.110.0.

Comment (zecakeh) [reply README.md:64]: The idea here is to have the same version as in config.toml. The goal of both is to declare the minimum supported version of Hugo for the website.

Comment (zecakeh) [reply README.md:64]: What I'm saying is either we need to downgrade the version in config.toml, or we need to upgrade it in the README.

Comment (richvdh) [reply README.md:64]: Right, sorry, I misunderstood the purpose of the config.toml setting.

**Review by richvdh — APPROVED** [review]
Thanks!

**Pull request merged**